manage.py

```diff
@@ -132,27 +132,57 @@ def load_nicknames():
     """For improved search when candidates have a name that doesn't appear on their form.
     Additional nicknames can be added to the csv for improved search.
     """
+
+    logger.info('Loading nicknames...')
+
     import pandas as pd
     import sqlalchemy as sa
+
     try:
-        table = sa.Table('ofec_nicknames', db.metadata, autoload_with=db.engine)
+        table = sa.Table(
+            'ofec_nicknames',
+            db.metadata,
+            autoload_with=db.engine
+        )
         db.engine.execute(table.delete())
     except sa.exc.NoSuchTableError:
         pass
-    load_table(pd.read_csv('data/nicknames.csv'), 'ofec_nicknames', if_exists='append')
+
+    load_table(
+        pd.read_csv('data/nicknames.csv'),
+        'ofec_nicknames',
+        if_exists='append'
+    )
+
+    logger.info('Finished loading nicknames.')
 
 @manager.command
 def load_pacronyms():
     """For improved search of organizations that go by acronyms
     """
+
+    logger.info('Loading pacronyms...')
+
     import pandas as pd
     import sqlalchemy as sa
+
     try:
-        table = sa.Table('ofec_pacronyms', db.metadata, autoload_with=db.engine)
+        table = sa.Table(
+            'ofec_pacronyms',
+            db.metadata,
+            autoload_with=db.engine
+        )
         db.engine.execute(table.delete())
     except sa.exc.NoSuchTableError:
         pass
-    load_table(pd.read_excel('data/pacronyms.xlsx'), 'ofec_pacronyms', if_exists='append')
+
+    load_table(
+        pd.read_excel('data/pacronyms.xlsx'),
+        'ofec_pacronyms',
+        if_exists='append'
+    )
+
+    logger.info('Finished loading pacronyms.')
 
 def load_table(frame, tablename, if_exists='replace', indexes=()):
```

Comment:
Not something to be addressed in this PR but this reminds me we can delete this code.